docs/development_suite/set-up-infrastructure/project-settings.md

```diff
@@ -0,0 +1,116 @@
+
+The above information can be edited by clicking on the "Edit" button, which will open the modal below. It will then be sufficient to edit the fields to be updated and click the "Save changes" button to implement the change.
+
+![edit runtime settings](img/settings-edit-runtime-settings.png)
+
+### Security Features
+
+![security features](img/settings-security-features.png)
+
+This card shows and gives the user the ability to change some security features, provided as pod annotations, pod specs and Security Context. Specifically:
+- **Seccomp Profile**: if enabled, is enables the `seccompProfile` attribute of the `securityContext` object in order to restrict a Container's syscall. This feature is available as of Kubernetes v1.19 and you can learn more by taking a look at the [official Kubernetes documentation](https://kubernetes.io/docs/tutorials/security/seccomp/).
+- **AppArmor**: it is a Linux Security Module that, if enabled, implements Mandatory Access Control since [Kubernetes v1.4](https://kubernetes.io/docs/tutorials/security/apparmor/). AppArmor annotations set up a profile used by containerd to harden containerized applications to contrain exploitation. The template for such profile is available [on GitHub](https://github.com/moby/moby/blob/master/profiles/apparmor/template.go).
+- **Host Properties**: if enabled, it sets to `false` the following parameters, which could be used to allow access to shared information and elevate privileges:
+  * `hostPID`: controls whether containers can share host process namespaces;
+  * `hostIPC`: enables to read the shared memory between processes that communicate with IPC mechanisms;
+  * `hostNetwork`: controls whether containers can use the host network and allows to bypass network policies.
+
+  These fields are described in the [Pod Security Policies section of the official Kubernetes documentation](https://kubernetes.io/docs/concepts/security/pod-security-policy/).
+- **Privileged Pod**: if enabled, it configures the following attributes in a `securityContext` object:
+  * `allowPrivilegeEscalation`: controls whether a process can gain more privileges than its parent process, by default is set to true when the `CAP_SYS_ADMIN` capability is enabled or is run as privileged;
+  * `privileged`: controls whether the Pod can run privileged containers.
+
+  You can learn more about the `securityContext` object by taking a look at the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/).
+
+The above information can be edited by clicking on the "Edit" button, which will open the modal below, through which you can turn on/off the security features. Upon clicking "Save changes", the information will be updated.
+
+![edit security features](img/settings-edit-security-features.png)
+
+## Early Access Features
+
+The "Early access features" tab allows the user to visualize and activate features in early access, through a single homonymous card.
```

Comment:
specificherei che le feature in preview saranno attivate e disabilitate **solo sul progetto in questione**. Questo perchè esistono le features anche sul singolo utente e magari crea confusione